tests/test_camera.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+from unittest.mock import patch
+import sense.camera as camera
+import os
+import cv2
+
+
+class TestGetImage(unittest.TestCase):
```

Comment:
You can rename this class to `TestVideoSource` and include all the methods in the other classes that are testing `VideoSource` -- all of them use the same `setup()` function. 

The same could then be done with a class for `TestVideoStream` and `TestVideoWriter`

Reply:
Thanks, I've updated them.